### ISSUE TITLE
Unhelpful Log message

### ISSUE THREAD

**Issue by IRF160** [input-output-hk/mithril#546]
## Context & versions
following the build version of mithril-signer 10/11/2022 

## Steps to reproduce
  1. compiled the new mithril-signer with experiential mode enabled
  2. Start the mithril-signer
  3. view syslogs through journalctl 

## Actual behavior
Error message in the logs
{"msg":"STATE MACHINE: an error occured: ","v":0,"name":"slog-rs","level":50,"time":"2022-10-12T02:10:54.879078525Z","hostname":"relay25t","pid":21503,"error":"RemoteServerLogical(\"bad request\")"}

## Expected behavior
more details needed instead of "bad request" 

Arnaud looked on the mithril-aggregator with an error of "KesSignatureInvalid" which would be more helpful

**Comment by Alenar** [comment]
@IRF160 We just merged PR #547, it enhance some log messages (including the one for a failed signer registration), in your case with an updated signer for the same error you would now see:
```json
{
    "msg": "STATE MACHINE: an error occured: ", "v": 0, "name": "slog-rs", "level": 50, "time": "2022-10-12T02:10:54.879078525Z", "hostname": "relay25t", "pid": 21503,
    "error": "RemoteServerLogical(\"bad request: {\"label\":\"failed_signer_registration\",\"message\":\"KesSignatureInvalid\"}\")"
}
```
Is that ok for you ? If so we will close this issue.

**Comment by IRF160** [comment]
This solution is good with me
